docs/SETUP.md

```diff
@@ -149,7 +149,12 @@ $ cd ${SPLUNK_HOME}/etc/apps
 ```
 
 ### First Run
-If SA-Eventgen App is correctly installed, there is no additional configuration required. SA-Eventgen app will automatically identify with any apps with eventgen.conf.
+If your `SA-Eventgen` is 7.0+, you need to add a new parameter under `[general]` stanza in `$SPLUNK_HOME/etc/system/local/server.conf`:
+```
+[general]
+python.version = python3
+```
+Then `SA-Eventgen` app will automatically identify with any apps with `eventgen.conf`.
```

Comment:
Currently the latest splunk build seems to have bugs when place this in to `inputs.conf`.

Reply:
This bug is fixed in core now.